docs/resources/openstack_vm.md

```diff
@@ -0,0 +1,753 @@
+# CERN OpenStack Virtual Machines
+
+## Introduction
+
+CERN provides an Infrastructure-as-a-Service as part of their private cloud, which any member can access.
+Through [OpenStack][openstack]{target=_blank} CERN allows users to create virtual machines on their computing infrastructure.
+Using self-service portals one can rapidly request virtual machines for production, testing and development purposes, accessible through an ssh connection.
+
+The machines can be of different capacities and run a variety of Windows or Linux operating systems.
+This page provides a quick setup guide suitable for OMC members to create their own lxplus-like virtual machine on CERN OpenStack
+and is based on the [OpenStack CS8 guide for ABP members][openstackCS8]{target=_blank}.
+
+For more details, see both the [CERN OpenStack][cern_openstack]{target=_blank .cern_login} website and its [documentation][cern_openstack_doc]{target=_blank} pages.
+
+### What to expect
+
+After following this guide you will have your own virtual machine running on CERN OpenStack.
+This machine will be similar to your office desktop or the `lxplus` machines,
+with access to `afs`, `eos` and `htcondor` yet come with some advatages and disadvantages:
+
+**Advantages:**
+
+- Full root access to your machine.
+- Independence from Building 6/9/10 powercuts (in comparison to your office desktop machine).
+- Customizable hardware (RAM, Disk, CPU).
+- Persistent storage (your machine will stay alive until you delete it).
+- No interference from other users (your machine is yours alone).
+- Ability to install custom software.
+- Ability to run long-running processes without being logged in.
+- Ability to restart the machine whenever needed.
+- Large shareable local storage (if you mount a volume).
+
+**Disadvantages:**
+
+- No automatic updates (you have to maintain your machine yourself).
+- No external access, only via CERN network or lxplus-tunneling.
+- Not a high-performace machine (compared to dev/optics servers and HPC clusters), especially when not part of a [project](#projects).
```

Comment:
I would remove the anchor here. Firstly because it links to the next word, and more importantly because the next sentence also has a `project` word that links to a different place (and that one is important).